code-of-conduct-transparency-reports/2019-07-02-coc-transparency-report.md

```diff
@@ -0,0 +1,29 @@
+# The Carpentries Code of Conduct Transparency Report
+## 2019 July 02
+
+## Overview
+
+### Reports
+
+Between 2018-11-09 and 2019-07-09, the Code of Conduct committee has not received an incident report, although the [Incident Reporting Form](https://docs.google.com/forms/d/e/1FAIpQLSdi0wbplgdydl_6rkVtBIVWbb9YNOHQP_XaANDClmVNu0zs-w/viewform) was submitted once with no information included.
+
+### Potential Code of Conduct Breaches  
+There have been no officially reported Code of Conduct breaches in the last eight months. However, discussions in the community and within the Code of Conduct committee have led the Executive Council to approve a task force that will develop recommendations for incidents that fall outside the scope of the Code of Conduct committee. 
+
```

Comment:
I suspect that @kcranston would like to see a change to

"that fall outside the scope of the Code of Conduct", 

The thing that has a scope is the CoC, not the CoCc. What we do is dictated by the CoC, not the other way around.

Reply:
The goal of the task force is _to make recommendations for guidelines, approaches, support structures or policies that could be developed to respond to incidents that happen outside the scope of the Carpentries Code of Conduct committee._ We've seen many instances where the CoCc has no duty to act, but something _**could**_ be done. This task force will make recommendations that _**may**_ potentially broaden the scope of the CoCc.

Reply:
I have a few comments here: First, it is the CoC that has scope, not the committee as such (see comment above). 

Also, could I suggest replacing the phrasing "has no duty to act" with "has no mandate to act", maybe? Might just be my reading of it, but "no duty" could easily be read as "we don't bother", whereas the actual situation is that we cannot act, whether we'd want to or not.

Reply:
Thanks all. The goal of the transparency report is to note past events. As the Task Force is still being scoped, I think exactly what it is doesn't need to be included here. So, it can be language that just notes that a task force is being convened. Would this be appropriate for the transparency report? 

"
However, discussions in the community and within the Code of Conduct committee have led the Executive Council to approve a relevant task force.
" 
 
Then task force language can be handled separately.

Reply:
I agree that the task force language should be handled separately in a different place. This report needs to include information about reported incidents only.

Reply:
Agreed! I'll update the language.